.editorconfig

```diff
@@ -16,5 +13,19 @@ fsharp_max_array_or_list_width=120
 fsharp_max_infix_operator_expression=80
 fsharp_max_value_binding_width=120
 
+# Specific settings for "View/HTML" related files
+# It makes the code more consistent in term of spacing and indentation
 [GenerateHtml.fs]
-fsharp_experimental_elmish = true
\ No newline at end of file
+fsharp_experimental_elmish = true
+fsharp_array_or_list_multiline_formatter = number_of_items
+fsharp_max_array_or_list_number_of_items = 0
+fsharp_multi_line_lambda_closing_newline = true
+fsharp_multiline_bracket_style = aligned
+fsharp_keep_max_number_of_blank_lines = 1
+fsharp_align_function_signature_to_indentation = true
+# Keep the if/then/else consistent
+fsharp_max_if_then_else_short_width = 0
```

Comment:
I don't think this really solves any problem.

Reply:
This solves consistency in formatting:

![CleanShot 2025-01-02 at 11 04 30@2x](https://github.com/user-attachments/assets/7adc46b3-23b9-47b3-ac0a-1d7bc0e73982)

Personally, I value having a single way of reading the code compared to have sometimes something take 1 lines, sometimes 2 or more.

Can disable if still wanted

Reply:
Please remove